src/features/notifications/NotificationList.tsx

```diff
@@ -49,50 +84,86 @@ const NotificationList = ({ notifications, refreshing, onRefresh }: Props) => {
+          addSuffix: true,
+        }),
+        data: grouped[k].sort((a, b) => b.time - a.time),
+      }))
+      .sort((a, b) => b.data[0].time - a.data[0].time)
 
     animateTransition('NotificationList.SortedAndGrouped')
     setGroupedNotifications(arr)
-  }, [allNotifications])
+  }, [allNotifications, getNotificationGroupTitle, t])
+
+  const loadMoreNotifications = useCallback(() => {
+    dispatch(fetchMoreNotifications(notifications[notifications.length - 1].id))
+  }, [dispatch, notifications])
 
   return (
-    <Box flex={1} alignContent="space-between">
+    <Box flex={1} marginBottom="m">
       <Text variant="h3" marginVertical="m" flexGrow={1} paddingHorizontal="l">
         {t('notifications.list.title')}
       </Text>
 
-      <FlatList
-        inverted
-        refreshControl={
-          <RefreshControl
-            refreshing={refreshing}
-            onRefresh={onRefresh}
-            tintColor="white"
-          />
-        }
-        refreshing={refreshing}
-        style={{ flexGrow: 0 }}
-        data={groupedNotifications}
-        keyExtractor={(item) => item[0].id.toString()}
-        renderItem={({ item }) => (
-          <NotificationGroup
-            notifications={item}
-            onNotificationSelected={setSelectedNotification}
-          />
-        )}
-      />
-      <NotificationShow
-        notification={selectedNotification}
-        onClose={() => setSelectedNotification(null)}
-      />
+      <Box
+        backgroundColor="white"
+        borderRadius="xl"
+        marginBottom="xl"
+        overflow="hidden"
+      >
+        <SectionList
+          stickySectionHeadersEnabled={false}
+          contentContainerStyle={{
+            paddingHorizontal: spacing.l,
+            paddingTop: spacing.l,
+          }}
+          scrollIndicatorInsets={{
+            top: spacing.m,
+            bottom: spacing.m,
+            left: 0,
+            right: 0,
+          }}
+          refreshControl={
+            <RefreshControl
+              refreshing={refreshing}
+              onRefresh={onRefresh}
+              tintColor={colors.grayLight}
+            />
+          }
+          refreshing={refreshing}
+          style={{ flexGrow: 0 }}
+          sections={groupedNotifications}
+          keyExtractor={(item) => item.id.toString()}
+          renderItem={({ item, index, section }) => (
+            <NotificationItem
+              onNotificationSelected={setSelectedNotification}
+              notification={item}
+              isLast={index === section.data.length - 1}
+            />
+          )}
+          renderSectionHeader={({ section: { title, icon, time } }) => (
+            <NotificationGroupHeader title={title} icon={icon} time={time} />
+          )}
+          onEndReached={loadMoreNotifications}
+          onEndReachedThreshold={0.2}
+        />
+        <NotificationShow
+          notification={selectedNotification}
+          onClose={() => setSelectedNotification(null)}
```

Comment:
this one too

Reply:
It would be nice to add a close handler when you click on the background of this `<NotificationShow`. 

I think you could just add a TouchableOpacity to it. Something like...
```ts
    <Modal
      presentationStyle="overFullScreen"
      transparent
      visible={!!notification}
      onRequestClose={onClose}
      animationType="fade"
    >
      <Box flex={1} justifyContent="flex-end" flexDirection="column">
        <BlurBox
          top={0}
          left={0}
          bottom={0}
          right={0}
          blurAmount={70}
          blurType="dark"
          position="absolute"
        />
        <TouchableOpacityBox flex={1} onPress={onClose} />

        <Card variant="modal" padding="l" margin="m"
```

Reply:
The show view feels strange being fixed to the bottom now too. Doesn't look like there's a new design in Figma though...

Reply:
Yeah I agree, maybe we move it to the middle?

Reply:
I moved it to the top and put the dismiss below it, feels better I think